rubicon-htb.js

```diff
@@ -254,6 +256,40 @@ function RubiconModule(configs) {
         return firstPartyData;
     }
 
+    function _getDigiTrustQueryParams() {
+        if (configs.digitrustId) {
+            _digiTrustId = configs.digitrustId;
+        }
+            function getDigiTrustId() {
+                if (!Browser.isTopFrame()) {
+                    try {
+                    var _window = window.top;
+                    } catch(e) {
+                        console.log("impossible to reach top window, get topmost accessible window context  ");
+                        var _window = Browser.topWindow;
+                    }
+                } else {
+                    var _window = window;
+                }
+                try {
+                    var digiTrustUser =  (_digiTrustId || _window.DigiTrust.getUser({member: 'T9QSFKPDN9'}))
```

Comment:
for efficiency, most of this logic is not needed if `_digiTrustId` is already set.   you could either check this at the beginning of the `getDigiTrustId()` function.  `if (_digiTrustId) return _digiTrustId;` or not even call `getDigiTrustId()` at all (see comment below).